### PR TITLE
docs(spinner): additional fixes

### PR DESCRIPTION
**Pull request by nikkimk**
## What I did

1. Updated spinner docs based on reopened #904 [feedback](https://github.com/RedHat-UX/red-hat-design-system/issues/904#issuecomment-1569079659).

## Testing Instructions

1. View the [feedback](https://github.com/RedHat-UX/red-hat-design-system/issues/904#issuecomment-1569079659).
2. View [DP spinner docs](https://deploy-preview-1013--red-hat-design-system.netlify.app/elements/spinner/).
3. Compare to [spinner doc](https://docs.google.com/document/d/1qqH_TFsoaeOmmg-515LtpEq6putJOZjPLbhfVqoDrDY/edit#heading=h.l1zi1rirh1yb) and to [Docs cleanup checklist](https://github.com/RedHat-UX/red-hat-design-system/wiki/Docs-Cleanup-Q2-2023-Checklist).

## Notes to Reviewers
The following are WIP to be addressed as other issues/PRs:
- [Image width styling](https://github.com/RedHat-UX/red-hat-design-system/pull/891)
- [Code tab styling](https://github.com/RedHat-UX/red-hat-design-system/issues/969)

### CODE REVIEW

File: elements/rh-spinner/docs/10-style.md [hunk 1]
```diff
@@ -9,9 +9,12 @@ A spinner is an animated line segment that follows a track and may include an op
           alt=" Anatomy of a spinner with annotations; number 1 is pointing to the track, number 2 is pointing to the indicator, and number 3 is pointing to the optional text label",
           src="../spinner-anatomy.png" %}
 
+<br>
```
Comment (bennypowers) [inline elements/rh-spinner/docs/10-style.md:12]: shouldn't need this, but might need to add a single newline inbetween the last list item and the `{.example-notes}`

Comment (bennypowers) [reply elements/rh-spinner/docs/10-style.md:12]: see https://www.npmjs.com/package/markdown-it-attrs#ambiguity

File: elements/rh-spinner/docs/10-style.md [hunk 2]
```diff
@@ -80,6 +90,9 @@ If a small size spinner is used within a button, it can be positioned to the lef
 
 | Spacer | Current value |
 | ------ | ------------- |
-| ![8px spacer]({{ '../spinner-16px-spacer.png' | url }}) | 16px |
+| ![8px spacer]({{ '../spinner-16px-spacer.png' | url }}){style="width:16px;"} | <code>16px</code> |
```
Comment (bennypowers) [inline elements/rh-spinner/docs/10-style.md]:
```suggestion
| ![8px spacer]({{ '../spinner-16px-spacer.png' | url }}){width=16} | `16px` |
```

File: elements/rh-spinner/docs/10-style.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ A spinner comes in large, medium, and small sizes. Each size includes an optiona
 <br>
 
 | Size | Element | Current value |
-| ---- | ------- | ------------- |
+| ------- | ------- | ------- |
```
Comment (bennypowers) [inline elements/rh-spinner/docs/10-style.md:33]: if you want to automatically format these, you might like 
https://learn.microsoft.com/en-us/contribute/docs-authoring/reformat-table
or 
https://marketplace.visualstudio.com/items?itemName=pharndt.vscode-markdown-table

File: elements/rh-spinner/docs/00-overview.md
Comment (bennypowers) [inline elements/rh-spinner/docs/00-overview.md:1]: reserving comment for design approval

**Comment by coreyvickery** [comment]
@nikkimk So close!

### Style tab
- Under the `Sizes` heading, in the table under `Current value`, the values are appearing on the same line for me, can we add one more `<br />`?

### Guidelines tab
- Under the `Sizes` heading, the image should not be in a white container

That is it, two tiny changes.

File: elements/rh-spinner/docs/00-overview.md
Comment (bennypowers) [reply elements/rh-spinner/docs/00-overview.md:1]: @coreyvickery try now, and resolve this conversation if it fits.

**Review by coreyvickery — APPROVED** [review]
@bennypowers Thanks, looks good now.

**Pull request merged**